### DISCUSSION TITLE
Changes not appearing in unstaged area

### DISCUSSION THREAD

**Discussion by MattAnthonyWard** [atom/atom#22704] [Q&A]
Saved edits to my Python file in Atom cause the filename to change colour in the leftmost project window but nothing is appearing in the unstaged changes area - is there a reason why? I've set my folder up as a repo and made an initial and one further commit (this was done on a couple of occasions when the changes did show) but most of the time nothing is showing up. Any help appreciated.

**Comment by darangi** [comment]
What do you see when you execute `git status` in your project directory?
